doc/classes/SplitContainer.xml

```diff
@@ -24,6 +24,13 @@
 		<member name="dragger_visibility" type="int" setter="set_dragger_visibility" getter="get_dragger_visibility" enum="SplitContainer.DraggerVisibility" default="0">
 			Determines the dragger's visibility. See [enum DraggerVisibility] for details.
 		</member>
+		<member name="push_parent" type="bool" setter="set_push_parent" getter="get_push_parent" default="false">
+			If [code]true[/code], the [SplitContainer] will adjust its parent's [member split_offset] when dragging has reached the child's minimum size so dragging can continue. This can be chained in multiple [SplitContainer]s.
+			[b]Note:[/b] If the parent [SplitContainer] is not aligned with [member vertical], its parent will be used instead. If the parent is not a [SplitContainer], nothing happens.
+		</member>
+		<member name="resize_separately" type="bool" setter="set_resize_separately" getter="get_resize_separately" default="false">
+			If [code]true[/code], the [SplitContainer] will adjust its child's [member split_offset] when dragging so only the closest split gets resized, until prevented by the child's minimum size. If [member push_parent] is [code]false[/code], dragging cannot continue past its descendent [SplitContainer]s [member split_offset] if they align [member vertical]ly.
```

Comment:
Well if `push_parent` affects child containers, it should be renamed. Maybe `drag_push` or something, sounds very generic.

Reply:
Also I don't think so we should pun so heavily with `[member vertical]ly`, that'll be a translator's nightmare

Reply:
Would `push_nested` work better? If not I can use `drag_push`.

Reply:
`push_nested` sounds fine too.